src/charm.py

```diff
@@ -51,7 +70,7 @@ def __init__(self, *args):
         self._init_postgresql_relation()
 
     def _get_pebble_config(self, event: ops.framework.EventBase) -> dict:
```

Comment:
I think this should be renamed to `_get_gunicorn_pebble_config` for consistency.